Instance panel properties update.md (#174)
GDevelopApp/GDevelop-documentation: docs/gdevelop5/interface/scene-editor/index.md

An object that you put on the scene _is called **an instance**_. The Properties panel allows you to change the properties of the selected instance(s):

![Instance panel](https://github.com/GDevelopApp/GDevelop-documentation/assets/103995399/afce132f-fab7-42de-ae0c-b6f923affd0f)

>   1) **Name indicator:** displays the name of the object that the *instance* represents

Reviewer: This is the result:
<img width="802" alt="image" src="https://github.com/GDevelopApp/GDevelop-documentation/assets/1280130/ef9ce814-b4ff-4817-9cf9-a1e9d3f3b298">

It's not properly formatted because you should use `1.` instead of `1)`. See thuis guide: https://www.markdownguide.org/basic-syntax/#ordered-lists

>   2) **Instance lock:** blocks an instance's possition. It has 3 states: *ii* blocks the instance while maintaining its edition capabilites, *iii* blocks the instance's possition, edition *and* selection, and *i* unlocks the instance completley.

Reviewer: possition => position

I would have roman numeral here, not sure they are super obvious to the reader?

Reviewer: completley => completely

Another method to unlock an instance is finding the instance on the [instance panel list](/gdevelop5/interface/scene-editor/#objects-groups-panel) and clicking on the icon to unlock the desired instance
![Lock states](https://github.com/GDevelopApp/GDevelop-documentation/assets/103995399/62a5d956-e3ba-4dc5-88ca-9b5bc039ebc1)
  3) **Edit object:** opens the dialog to modify the instance's object

>   4) **Possition values:** instance's coordinates on the scene

Reviewer: ```suggestion
  4) **Position values:** instance's coordinates on the scene
```

  5) **Size values:** instance's size values like Width -horizontal size-, Height -vertical size- and Depth for 3D objects

>   6) **Mantain proportions:** links size values to be modified proportionally. Values can be modified by typing the size in the input, or by clicking on the input's name and dragging horizontally to increase or decrease the value

Reviewer: Maintain proportions (typo).

Or use "Keep ratio"

  7) **Reset to object values:** reset modified values in the instance to their object's original size

>   8) **Layer:** displays the location of the instance

Reviewer: "Location" is not precise enough:

```suggestion
  8) **Layer:** the layer on which the instance lives
```

>   9) **Animation:** displays the instance's animation for tile management

Reviewer 2: Here is the preview: https://gdevelop-wiki-git-fork-lunimoon-patch-6-gdevelop.vercel.app/gdevelop5/interface/scene-editor/#instance-properties-panel

To make it work, I think you should use `1.` notation without adding indent.

>   9) **Animation:** displays the instance's animation for tile management

Reviewer: ```suggestion
  9) **Animation:** the initial animation shown by the instance
```
